pkg/identity-manager/handlers.go

```diff
@@ -72,3 +111,26 @@ func (h *Handlers) redirect(params operations.RedirectParams, principal interfac
 	location := fmt.Sprintf("%s?%s", redirect, values.Encode())
 	return operations.NewRedirectFound().WithLocation(location)
 }
+
+func getRequestAttributes(request *http.Request) *attributesRecord {
+	log.Debug("Headers: %s\n", request.Header)
+	// Retrieve the original request method and map REST verb to policy operations
+	requestMethod := request.Header.Get("X-Original-Method")
+	operation := ""
+	switch requestMethod {
+	case "GET":
+		operation = "get"
```

Comment:
If this were Java I would recommend making `operation` into an enum value. Not sure how golang handles this, but an Enum makes sure that someone doesn't misspell the operation name causing a bug in the future. Not worth bending over backwards if golang doesn't support an analogous construct.

Reply:
This could be addressed with constants from `http` package: https://golang.org/pkg/net/http/#pkg-constants

Reply:
@tenczar Yeah makes sense. Thanks @kars7e I will use the http pkg constants.